tests/integration/example_test_env_vars.yaml

```
@@ -33,3 +33,9 @@ get:
   - ["golang.org/x/tools", "gomod", "v0.0.0-20190325161752-5a8dccf5b48a"]
   - ["gopkg.in/check.v1", "gomod", "v1.0.0-20180628173108-788fd7840127"]
   - ["gopkg.in/yaml.v2", "gomod", "v2.2.2"]
+# Test data for the Dependency replacement test
+dep_replacement:
+  # The list of dependencies that are replaced with specific versions
+  dependency_replacements:
+  - ["github.com/pkg/errors", "gomod", "v0.8.0"]
+  - ['github.com/Masterminds/semver', 'gomod', 'v1.4.0']
```

Rather than modify the current structure into what Cachtio expects in the payload, how about just setting it that way from the beginning? It's also easier to understand.
```suggestion
  - name: github.com/pkg/errors
    type: gomod
    version: v0.8.0
  - name: github.com/Masterminds/semver
    type: gomod
    version: v1.4.0
```